rollup.config.js

```diff
@@ -10,7 +10,7 @@ const config = {
     format: "umd",
     indent: false,
     extend: true,
-    banner: `// ${meta.homepage} v${meta.version} Copyright ${(new Date).getFullYear()} ${meta.author.name}`,
+    banner: `// ${meta.homepage} v${meta.version} Copyright ${(new Date(process.env.SOURCE_DATE_EPOCH ? (process.env.SOURCE_DATE_EPOCH * 1000) : new Date().getTime())).getFullYear()} ${meta.author.name}`,
```

Comment:
Could be a bit simpler:
```suggestion
    banner: `// ${meta.homepage} v${meta.version} Copyright ${(process.env.SOURCE_DATE_EPOCH ? new Date(process.env.SOURCE_DATE_EPOCH * 1000) : new Date).getFullYear()} ${meta.author.name}`,
```

Reply:
```suggestion
    banner: `// ${meta.homepage} v${meta.version} Copyright ${(new Date(process.env.SOURCE_DATE_EPOCH * 1000 || Date.now()).getFullYear()} ${meta.author.name}`,
```

:p